autotest/pymod/test_cli_utilities.py

```diff
@@ -29,59 +29,14 @@
 ###############################################################################
 
 import os
-import sys
```

Comment:
it would be good if those changes to test_cli_utilities.py were put in a dedicated PR

Reply:
I didn't realize those were in there until after I pressed the button. Are you OK with them as they are?

Reply:
Actually, I'm afraid that's going to break Windows testing, because with MSVC builds, binaries go to $build_dir/apps/$config_name where config_name=Release, Debug, RelWithDebInfo, etc.
cmake/helpers/GdalSetRuntimeEnv.cmake tests care of correctly appending that directory in the PATH.

Reply:
Is there some reason not to drive all this from cmake and avoid all of this? cmake knows where it stuck the binaries.

Reply:
it is convenient to be able to run a single test file locally with pytest, without going through a whole test suite